app.go

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
```

Comment:
this file shouldn't exist, its content should be in `main.go`.
Make sure you use `ignite app scaffold` to scaffold an app and follow the recommended app structure.